### PR TITLE
[requestVideoFrameCallback article] Improve title

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Improve the title. The current one wraps very unfortunately:
  Screen Shot 2020-08-14 at 11 43 20

### CODE REVIEW

**Comment by kaycebasques** [comment]
Titles have huge impact on documentation SEO. I recommend finding a way to keep `requestVideoFrameCallback` in there somehow. How about:

* Perform efficient per-video-frame operations on video with `requestVideoFrameCallback()`
* Efficient per-video-frame operations with `requestVideoFrameCallback()`

Or perhaps removing the code formatting?

I think @jpmedley also feels strongly about keeping `()` when talking about functions/methods

File: src/site/content/en/blog/requestvideoframecallback-rvfc/index.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
 title: |
-  The `requestVideoFrameCallback()` method
+  Perform efficient per-video-frame operations on video
```
Comment (tomayac) [inline src/site/content/en/blog/requestvideoframecallback-rvfc/index.md:3]:
```suggestion
  Perform efficient per-video-frame operations on video with `requestVideoFrameCallback()`
```

File: src/site/content/en/blog/requestvideoframecallback-rvfc/index.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 subhead: |
-  Perform efficient per-video-frame operations on video.
+  Learn how to use the `requestVideoFrameCallback()` to work more efficiently with videos in the browser.
 authors:
   - thomassteiner
 date: 2020-06-29
-updated: 2020-06-29
+updated: 2020-08-14
```
Comment (tomayac) [inline src/site/content/en/blog/requestvideoframecallback-rvfc/index.md:9]:
```suggestion
updated: 2020-08-17
```

**Comment by tomayac** [comment]
Alright, I have changed to "Perform efficient per-video-frame operations on video with requestVideoFrameCallback()". It's just a long word, there's no way around it. PTAL. Thanks!

**Pull request merged**